controllers/usersController.js

```diff
@@ -73,7 +73,7 @@ class UsersController {
   static async sendPasswordRecoveryLink(request, response, next) {
     try {
       const { email } = request.body;
-      const token = jwt.sign({ email }, env('APP_KEY'), { expiresIn: '1h' });
+      const token = jwt.sign({ email }, env('PASSWORD_RESET_KEY'), { expiresIn: '1h' });
```

Comment:
Hi @valenstical You doing great. Just to clarify, why is there additional secret key for encrypting the email payload, whereas an `APP_KEY` is already available?

Reply:
Hello @devdbrandy , 
Due to the sensitivity involved in resetting a password, it is advisable to use a different private key with JWT. This way,  a compromise on the jswon web token from other parts of the application using JWT with the  APP_KEY will not have any negative effect on resetting password.

Reply:
Okay, that makes lot of sense. Great job @valenstical.